### PR TITLE
[docs] Add alt text on images (related to #4036)

### PR DESCRIPTION
**Pull request by akshitadixit**
This pull request mentions #4028 and modifies the following files only:

- [x] docs/Features.rst
- [x] docs/GPU-Performance.rst
- [x] docs/GPU-Windows.rst
- [ ] docs/Parallel-Learning-Guide.rst
- [ ] docs/Parameters-Tuning.rst
- [ ] docs/gcc-Tips.rst
- [ ] docs/index.rst

### CODE REVIEW

**Comment by jameslamb** [comment]
I apologize for the inconvenience, but could you please merge in the latest changes from `master`? We recently merged some fixes for failing continuous integration jobs (#4032)

**Comment by akshitadixit** [comment]
Sure

**Comment by akshitadixit** [comment]
@jameslamb if you could please let me know if the checks are failing on my end?

**Comment by akshitadixit** [comment]
Definitely, thanks for informing.

Also I came across these articles and then chose full stops instead of simple spaces. Also tried it out with a tts to confirm.
* [article](https://accessibility.blog.gov.uk/2017/02/08/advice-for-creating-content-that-works-well-with-screen-readers/#:~:text=A%20screen%20reader%20might%20speak,.%22%20is%20an%20interesting%20case)
* [article](https://accessibility.psu.edu/abbreviations/)

File: docs/GPU-Windows.rst
Comment (jameslamb) [inline docs/GPU-Windows.rst:414]:
```suggestion
   :alt: A screenshot of the command prompt where a segmentation fault has occurred while using Light G B M
```

I make this mistake a lot in this project haha

File: docs/GPU-Windows.rst
Comment (jameslamb) [inline docs/GPU-Windows.rst:381]:
```suggestion
      :alt: A screenshot of the git bash window with Light G B M successfully installed
```

File: docs/GPU-Windows.rst
Comment (jameslamb) [inline docs/GPU-Windows.rst:280]:
```suggestion
   :alt: A screenshot of the binary distributions of C Make for downloading on 64 bit Windows
```

I think the way you separated this out into `C` and `Make` will be better understood by screen readers.

File: docs/GPU-Performance.rst
Comment (jameslamb) [inline docs/GPU-Performance.rst:166]:
```suggestion
   :alt: A performance chart which is a record of the wall clock time after 500 iterations on G P U for Higgs, epsilon, Bosch, Microsoft L T R, Expo and Yahoo L T R and bin size of 63 performs comparitively better.
```

Based on this article you shared (https://accessibility.psu.edu/abbreviations/, thanks for sharing that!), I think that for these `alt` tags all acronyms should just use spaces and not full stops.

> When writing ALT tags with acronyms, add spaces between characters (e.g. <alt="P S U" >).

I also think that for this particular one, the "GPU" part should be moved to the middle of the sentence so it's clear that "AGPU" is not a meaningful acronym.

File: docs/GPU-Windows.rst
Comment (jameslamb) [inline docs/GPU-Windows.rst:111]:
```suggestion
   :alt: A screenshot of the Min G W installation setup settings window
```

File: docs/GPU-Windows.rst
Comment (jameslamb) [inline docs/GPU-Windows.rst:130]:
```suggestion
   :alt: A screenshot of the administrator command prompt where G C C version is being checked 
```

**Review by jameslamb — CHANGES_REQUESTED** [review]
Thanks so much for taking the time to do this, and for your patience with me as it took a long time for me to come back and give you a review.

Could you please consider the suggestions I just left? You can click "add to batch" in the browser and then "commit" after adding all of them to directly commit them, if you agree.

Can you also either merge in the latest `master` to this branch? Sorry for the inconvenience. We've had some issues recently with our automated continuous integration tests, and recent pull requests fixed those issues (I hope haha).

**Comment by akshitadixit** [comment]
Sure! Thanks for the suggestions, will do it.

File: docs/GPU-Windows.rst
Comment (akshitadixit) [reply docs/GPU-Windows.rst:414]: Dang T_T

**Comment by jameslamb** [comment]
@akshitadixit it looks like this branch has some conflicts with recent changes on `master`. I'd be happy to fix them if you'd like (they aren't related to your code). Can I push to your branch?

**Comment by akshitadixit** [comment]
> @akshitadixit it looks like this branch has some conflicts with recent changes on `master`. I'd be happy to fix them if you'd like (they aren't related to your code). Can I push to your branch?

Sure, please do!

**Review by jameslamb — APPROVED** [review]
Thanks so much! These changes look great to me.

Are you interested in helping us to finish https://github.com/microsoft/LightGBM/issues/4028? I'd be grateful for another pull request that adds alt text for any images in `docs/Parallel-Learning-Guide.rst` and `docs/gcc-Tips.rst`, if you have time and interest.

**Comment by akshitadixit** [comment]
Oh definitely! Will finish it off.

File: docs/Features.rst
Comment (StrikerRUS) [inline docs/Features.rst:57]:
```suggestion
   :alt: A diagram depicting leaf wise tree growth in which only the node with the highest loss change is split and not bother with the rest of the nodes in the same level. This results in an asymmetrical tree where subsequent splitting is happening only on one side of the tree.
```

File: docs/GPU-Performance.rst
Comment (StrikerRUS) [inline docs/GPU-Performance.rst:166]:
```suggestion
   :alt: A performance chart which is a record of the wall clock time after 500 iterations on G P U for Higgs, epsilon, Bosch, Microsoft L T R, Expo and Yahoo L T R and bin size of 63 performs comparatively better.
```

File: docs/GPU-Windows.rst
Comment (StrikerRUS) [inline docs/GPU-Windows.rst:49]:
```suggestion
   :alt: A screenshot of the System option under System and Security of the Control Panel.
```

File: docs/GPU-Windows.rst
Comment (StrikerRUS) [inline docs/GPU-Windows.rst:56]:
```suggestion
   :alt: A screenshot of the System Properties window.
```

File: docs/GPU-Windows.rst
Comment (StrikerRUS) [inline docs/GPU-Windows.rst:63]:
```suggestion
   :alt: A screenshot of the Environment variables window with variable path selected under the system variables.
```

File: docs/GPU-Windows.rst
Comment (StrikerRUS) [inline docs/GPU-Windows.rst:111]:
```suggestion
   :alt: A screenshot of the Min G W installation setup settings window.
```

File: docs/GPU-Windows.rst
Comment (StrikerRUS) [inline docs/GPU-Windows.rst:130]:
```suggestion
   :alt: A screenshot of the administrator command prompt where G C C version is being checked.
```

File: docs/GPU-Windows.rst
Comment (StrikerRUS) [inline docs/GPU-Windows.rst:228]:
```suggestion
   :alt: A screenshot of the command prompt that ends with text that reads - updated 14621 targets.
```

File: docs/GPU-Windows.rst
Comment (StrikerRUS) [inline docs/GPU-Windows.rst:252]:
```suggestion
   :alt: A screenshot of the website to download git that shows various versions of git compatible with 32 bit and 64 bit Windows separately.
```

File: docs/GPU-Windows.rst
Comment (StrikerRUS) [inline docs/GPU-Windows.rst:280]:
```suggestion
   :alt: A screenshot of the binary distributions of C Make for downloading on 64 bit Windows.
```

File: docs/GPU-Windows.rst
Comment (StrikerRUS) [inline docs/GPU-Windows.rst:299]:
```suggestion
      :alt: A screenshot with a pop-up window that reads - Build directory does not exist, should I create it?
```

File: docs/GPU-Windows.rst
Comment (StrikerRUS) [inline docs/GPU-Windows.rst:304]:
```suggestion
      :alt: A screenshot that asks to specify the generator for the project which should be selected as Min G W makefiles and selected as the use default native compilers option.
```

File: docs/GPU-Windows.rst
Comment (StrikerRUS) [inline docs/GPU-Windows.rst:320]:
```suggestion
      :alt: A screenshot of the C Make window after clicking on the configure button.
```

File: docs/GPU-Windows.rst
Comment (StrikerRUS) [inline docs/GPU-Windows.rst:381]:
```suggestion
   :alt: A screenshot of the git bash window with Light G B M successfully installed.
```

File: docs/GPU-Windows.rst
Comment (StrikerRUS) [inline docs/GPU-Windows.rst:398]:
```suggestion
   :alt: A screenshot of the command prompt where a binary classification model is being trained using Light G B M.
```

File: docs/GPU-Windows.rst
Comment (StrikerRUS) [inline docs/GPU-Windows.rst:414]:
```suggestion
   :alt: A screenshot of the command prompt where a segmentation fault has occurred while using Light G B M.
```

File: docs/GPU-Windows.rst
Comment (StrikerRUS) [inline docs/GPU-Windows.rst:429]:
```suggestion
   :alt: A screenshot of the Light G B M folder with 1 folder and 3 files selected to be removed.
```

File: docs/GPU-Windows.rst
Comment (StrikerRUS) [inline docs/GPU-Windows.rst:437]:
```suggestion
   :alt: A screenshot of the Cache Entry popup where the name is set to CMAKE_BUILD_TYPE in all caps, the type is set to STRING in all caps and the value is set to Debug.
```

File: docs/GPU-Windows.rst
Comment (StrikerRUS) [inline docs/GPU-Windows.rst:444]:
```suggestion
   :alt: A screenshot of the C Make window after clicking on configure and generate.
```

File: docs/GPU-Windows.rst
Comment (StrikerRUS) [inline docs/GPU-Windows.rst:458]:
```suggestion
   :alt: A screenshot of the command prompt after the command above is run.
```

**Review by StrikerRUS — CHANGES_REQUESTED** [review]
Very nice and detailed descriptions!
Please check some typos and add dots at the end for the consistency with descriptions in other files.

**Comment by akshitadixit** [comment]
> Very nice and detailed descriptions!
> Please check some typos and add dots at the end for the consistency with descriptions in other files.

Done the needful.

**Comment by jameslamb** [comment]
Looks good to me! @StrikerRUS I didn't want to overstep and dismiss your blocking review, so you'll have to approve before this can be merged.

**Review by StrikerRUS — APPROVED** [review]
@akshitadixit Thanks a lot for the contribution that required a thoughtful research and for quick fixes!

**Pull request merged**
